src/Command/Command.php

```diff
@@ -0,0 +1,178 @@
+        } catch (\Exception $e) {
+            throw new \Exception(sprintf($e->getMessage()));
+        }
+
+        return $this->builder;
+    }
+
+    /**
+     * @param OutputInterface $output
+     * @param int             $start
+     * @param int             $max
+     *
+     * @return ProgressBar
+     */
+    protected function createProgressBar(OutputInterface $output, $start, $max)
+    {
+        if ($this->progressBar === null || $max != $this->progressBarMax) {
+            $this->progressBarMax = $max;
+            $this->progressBar = new ProgressBar($output, $max);
+            $this->progressBar->setFormat(' %percent:3s%% [%bar%] %current%/%max%');
+            $this->progressBar->start();
+        }
+    }
+
+    /**
+     * @return ProgressBar
+     */
+    protected function getProgressBar()
+    {
+        return $this->progressBar;
+    }
+
+    /**
+     * Print progress bar.
+     *
+     * @param OutputInterface $output
+     * @param int             $itemsCount
+     * @param int             $itemsMax
+     * @param string          $message
+     */
+    protected function printProgressBar(OutputInterface $output, $itemsCount, $itemsMax, $message)
+    {
+        $this->createProgressBar($output, 0, $itemsMax);
+        $this->getProgressBar()->clear();
+        $this->getProgressBar()->setProgress($itemsCount);
+        $this->getProgressBar()->display();
+        if ($itemsCount == $itemsMax) {
+            $this->getProgressBar()->finish();
+            $output->writeln('');
+        }
+    }
+
+    /**
+     * Custom message callback function.
+     *
+     * @param OutputInterface $output
+     */
+    public function messageCallback(OutputInterface $output)
+    {
+        return function ($code, $message = '', $itemsCount = 0, $itemsMax = 0) use ($output) {
+            if ($this->quiet) {
+                return;
+            } else {
+                if (strpos($code, '_PROGRESS') !== false) {
+                    if ($output->isVerbose()) {
+                        if ($itemsCount > 0) {
+                            $output->writeln(sprintf(' (%u/%u) %s', $itemsCount, $itemsMax, $message));
+
+                            return;
+                        }
+                        $output->writeln("<info>$message</info>");
+                    } else {
+                        if (isset($itemsCount) && $itemsMax > 0) {
+                            $this->printProgressBar($output, $itemsCount, $itemsMax, $message);
+                        } else {
+                            $output->writeln("$message");
+                        }
+                    }
+                } elseif (strpos($code, '_ERROR') !== false) {
+                    $output->writeln("<error>$message</error>");
+                } elseif ($code == 'TIME') {
+                    $output->writeln("<comment>$message</comment>");
+                } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method messageCallback uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/Narno/Cecil/pullRequest?prid=4722362)